e2e/tests/README.md

```diff
@@ -0,0 +1,40 @@
+e2e tests
+=========
+
+> tl; dr - `make localnet_up` and then `make test_e2e` 
+
+You can also click the `e2e-tests` button in the Tilt UI, which is handy during development.
+
+## Design & Architecture
+
+The test suite is located in `e2e/tests` and it contains a set of Cucumber Gherkin feature files and the associated Go tests to run them. This is a test file that lays out the BDD pattern using the client's help function as an example.
+
+### Implementation 
+
+`make test_e2e` sees any files named with the pattern `*.feature` in `e2e/tests` and runs them with [godog](https://github.com/cucumber/godog), the Go test runner for Cucumber tests. We indirectly depend on the LocalNet to be up and running for our test suite to use. We call commands to steer around a container by calling `kubectl exec` and targeting a pod in the cluster by name. We are able to pass commands to that client and then assert on the behavior of that program.
```

Comment:
```suggestion
`make test_e2e` sees any files named with the pattern `*.feature` in `e2e/tests` and runs them with [godog](https://github.com/cucumber/godog), the Go test runner for Cucumber tests. There is an indirect dependance on the LocalNet to be up and running for our test suite to use. We call commands to steer around a container by calling `kubectl exec` and targeting a pod in the cluster by name. We are able to pass commands to that client and then assert on the behavior of that program.
```

Try to avoid using `we` and make it more 3rd person.